tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -154,10 +156,10 @@ await Task.Run(async () =>
             {
                 var template = new AdaptiveCardTemplate(cardTemplate);
 
-                var hostData = new JsonObject
+                var hostData = new Windows.Data.Json.JsonObject
                 {
                     // TODO Add support to host theme in hostData
-                    { "widgetSize", JsonValue.CreateStringValue(WidgetSize.ToString().ToLowerInvariant()) }, // "small", "medium" or "large"
+                    { "widgetSize", Windows.Data.Json.JsonValue.CreateStringValue(WidgetSize.ToString().ToLowerInvariant()) }, // "small", "medium" or "large"
                 }.ToString();

```

Comment:
While we're getting rid of newtonsoft, can we also get rid of Windows.Data.Json?
```suggestion
                var hostData = new JsonObject
                {
                    // TODO Add support to host theme in hostData
                    { "widgetSize", JsonValue.Create<string>(WidgetSize.ToString().ToLowerInvariant()) }, // "small", "medium" or "large"
                }.ToString();
```
https://devblogs.microsoft.com/oldnewthing/20230102-00/?p=107632

Reply:
We would need to convert this to use System.Text.Json instead.

Reply:
Where we can, I think it's better to use System.Text.Json. I don't think it's worth writing our own Merge function to be compatible with System.Text.Json objects, but I think we can use it easily in these few lines of code (they're isolated since we're going to string at the end anyway) so I'd suggest doing so.